### DISCUSSION TITLE
[BUG] students can join different class, but use only the last one

### DISCUSSION THREAD

**Discussion by hasan-sh** [hedyorg/hedy#5080] [Implementation discussions]
**Describe the bug**
Currently, students can be in different classes, but the customization of the last class is always applied. This issue should tackle the following:
- allow inviting a student to different classes at the same time
  - now, if you invite `student1` to a `Class1`, and then you (or some other teachers) invite them to `Class2`, the invitation will fail because there's one invitation for `student1`.
  - allow multiple invitations.
- allow students to switch between classes they are in
  - in the menu bar show dropdown with all classes (or in the profile setting section)
  - and students can choose which one to use.

**Comment by Felienne** [comment]
To be honest, I am not sure for what scenarios this is useful, so I will do a bit of inquiry with tecchers first. I know some teachers wanted to use this as a "hack" for customizations per student, but it would be better to truly support that!  

Let's first make sure the state cannot happen, by giving a better error messages with a second invite and an error message with a second class joining (see #4811)
